### PR TITLE
update brand apis 

### PR DESCRIPTION
**Pull request by navin9584**

### CODE REVIEW

File: v1.0/frontend/SaralApp/multi-tenant-branding.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (dileep-gadiraju) [inline v1.0/frontend/SaralApp/multi-tenant-branding.json:1]: this file needs to be removed as u r fetching from APIs

File: v1.0/frontend/SaralApp/src/modules/common/components/HeaderComponent.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ class HeaderComponent extends Component {
             customHeaderTextStyle,
             headerButton,
             customBtnStyle,
+            titletextstyle,
```
Comment (dileep-gadiraju) [inline v1.0/frontend/SaralApp/src/modules/common/components/HeaderComponent.js:19]: these changes has to be reverted.

File: specs/v1/swagger-saral-apidoc.yaml [hunk 3]
```diff
@@ -515,6 +517,67 @@ paths:
         "400":
           description: "unsuccessful operation"
           
+  /brand:
+    post:
+      security:
+        - Bearer: []
+      tags:
+        - "Brand"
+      summary: "Create Brand for school"
```
Comment (dileep-gadiraju) [inline specs/v1/swagger-saral-apidoc.yaml:526]: this is for state not for school. @apurvabisht97 , please correct it

**Comment by apurvabisht97** [comment]
i will update

File: v1.0/frontend/SaralApp/src/utils/Strings.js [hunk 4]
```diff
@@ -81,5 +84,6 @@ export default strings = new LocalizedStrings({
         student_roll_length_error: 'RollNo. should be 7 digit',
         sat_string: 'SAT',
         pat_string: 'PAT',
+        welcome_up:'Welcome to Saral Scan App'
```
Comment (dileep-gadiraju) [inline v1.0/frontend/SaralApp/src/utils/Strings.js:87]: @navin9584 , rename this constant to welcome_text instead of welcome_up

**Review by dileep-gadiraju — APPROVED** [review]

**Pull request merged**